src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Install and set up for iOS development
+description: Configure your system to develop Flutter for iOS.
+short-title: Set up iOS
+---
+
+## Install Flutter
+
+To set up your development environment for iOS,
+you first need to install Flutter on your
+development machine.
+
+If you have already installed Flutter,
+ensure that it's [up to date][].
+
+:::note Did you know?
+When you install or update the Flutter framework,
+it also installs and updates the Dart language for you.
+:::
+
+If Flutter is installed and up to date,
+continue to the next step.
```

Comment:
What do you think about reducing this section a bit? I believe most users navigating to this page should already have installed Flutter or even directly navigating to it from the Flutter set up page.

Combining yours with what I have for the WIP Android page:

```suggestion
Learn how to set up your development environment
to run, build, and deploy Flutter apps for iOS devices.

:::note
If you haven't set up Flutter already,
visit and follow the [Get started with Flutter][] guide first.

If you have already installed Flutter,
ensure that it's [up to date][].
:::

[Get started with Flutter]: /get-started/install
[up to date]: /install/upgrade
```